precompiles/werc20/events.go

```diff
@@ -0,0 +1,71 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package werc20
+
+import (
+	"math/big"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/vm"
+
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
```

Comment:
nit: group Geth imports
```suggestion
	"github.com/ethereum/go-ethereum/core/vm"
	ethtypes "github.com/ethereum/go-ethereum/core/types"

```